docs/release-notes/deprecate-containerRuntime.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**Deprecations**
+
+-  Singularity, Podman, and Apptainer Container runtimes for AgentRM: Launching a Singluarity/Podman/Apptainer container runtimes for Agent is no longer supported. Docker is the only option that is supported.
```

Comment:
To avoid repeating here I think I would start the note with 

``` AgentRM: ...```

I would add a note that reminds users
"If you want to use singularity, podman, or apptainer the Determined master enterprise edition still supports this <link to the launcher docs>"

I think I would also clarify that 
"This change only affects when ```container_runtime``` is set to ```podman```, using a podman emulation layer is unchanged"

Reply:
Thanks for suggesting! Its far better approach!